Makefile

```diff
@@ -19,3 +19,27 @@ test:
 	./scripts/wasmtestreport.py && \
 	cat results.json && \
 	! grep '"number_of_failures": [^0]' results.json
+
+.PHONY: lint
+lint:
+	cargo fmt --check --all --manifest-path src/wasmtime/Cargo.toml
+	cargo clippy \
+	    --manifest-path src/wasmtime/Cargo.toml \
+	    --all-features \
+	    --keep-going \
+	    -- \
+	    -A warnings \
+	    -A clippy::not_unsafe_ptr_arg_deref \
+	    -A clippy::absurd_extreme_comparisons
+
+.PHONY: format
+format:
+	cargo fmt --all --manifest-path src/wasmtime/Cargo.toml
+
+.PHONY: docs-publish
+docs-publish:
+	mkdocs gh-deploy --force 
```

Comment:
This already auto-updates when docs are merged into main, do we need this here? It seems dangerous